static/app/views/dashboardsV2/manage/dashboardCard.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import ActivityAvatar from 'app/components/activity/item/avatar';
+import Card from 'app/components/card';
+import Link from 'app/components/links/link';
+import {t} from 'app/locale';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import space from 'app/styles/space';
+import {User} from 'app/types';
+import {callIfFunction} from 'app/utils/callIfFunction';
+
+type Props = {
+  title?: string;
+  detail?: React.ReactNode;
+  to: object;
+  createdBy?: User | undefined;
+  dateStatus?: React.ReactNode;
+  onEventClick?: () => void;
+  renderWidgets: () => React.ReactNode;
+  renderContextMenu?: () => React.ReactNode;
```

Comment:
nit: can we move the optional props to the end of the object?

```suggestion
  to: object;
  renderWidgets: () => React.ReactNode;
  createdBy?: User | undefined;
  dateStatus?: React.ReactNode;
  onEventClick?: () => void;
  renderContextMenu?: () => React.ReactNode;
  title?: string;
  detail?: React.ReactNode;
```